data/chapters/austin.json

```diff
@@ -3,20 +3,20 @@
   "title": "Austin GiveCamp",
   "city": "Austin",
   "state": "Texas",
-  "latitude": 30.4010897,
-  "longitude": -97.7517582,
+  "latitude": 30.265886,
+  "longitude": -97.749298,
   "img": "austin.jpg",
   "img_attribution": "iStock",
   "events": [
     {
       "start_date": "2016-11-04",
       "end_date": "2016-11-06",
-      "street": "10415 Morado Circle Bld 3 Ste 100",
```

Comment:
Is this intended to be an address change on where the 2016 event was held?

Reply:
Yes, it's being moved to a new location.